docs/capabilities/objectives.md

```diff
@@ -2,7 +2,7 @@
 
 ## Objectives help you set direction and compare them with current state 
 
-**Objectives** are targets that can be set in Sigrid, to compare against system status and quality trends. It is considered separate from metadata (see [metadata page](../organization-integration/metadata.md)), since *Objectives* do not describe characteristics of systems themselves, but where you want them to be.
+**Objectives** are targets that can be set in Sigrid, to compare against system status and quality trends. In this context, objectives are understood as being *non-functional requirements*. It is considered separate from metadata (see [metadata page](../organization-integration/metadata.md)), since *Objectives* do not describe characteristics of systems themselves, but where you want them to be.
```

Comment:
Should we say:
" In this context, objectives are understood as being *non-functional requirements*. They are considered separate ....", given that we are talking about multiple objectives?